templates/helm/values.yaml.tpl

```
@@ -28,6 +28,9 @@ deployment:
   # Which priorityClassName to set?
   # See: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority
   priorityClassName: ""
+
+role:
+ labels: {}
```

Can we have different values for Role and ClusterRole?
```suggestion
role:
  labels: {}
clusterRole:
  labels: {}
```